go.mod

```diff
@@ -5,9 +5,11 @@ go 1.16
 require (
 	github.com/RHsyseng/operator-utils v1.4.2
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/ceph/go-ceph v0.11.0
```

Comment:
add dependency changes in a separate commit and run `make deps-update`.\

go.mod go.sum and vendor changes should be added in one commit.